astroquery/esa/jwst/tests/test_jwsttap.py

```diff
@@ -0,0 +1,1088 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+===============
+eJWST TAP tests
+===============
+
+European Space Astronomy Centre (ESAC)
+European Space Agency (ESA)
+
+"""
+import os
+import shutil
+from unittest.mock import MagicMock
+
+import astropy.units as u
+import numpy as np
+import pytest
+from astropy import units
+from astropy.coordinates.sky_coordinate import SkyCoord
+from astropy.table import Table
+from astropy.units import Quantity
+
+from astroquery.esa.jwst import JwstClass
+from astroquery.esa.jwst.tests.DummyTapHandler import DummyTapHandler
+from astroquery.ipac.ned import Ned
+from astroquery.simbad import Simbad
+from astroquery.utils import TableList
+from astroquery.utils.tap.conn.tests.DummyConnHandler import DummyConnHandler
+from astroquery.utils.tap.conn.tests.DummyResponse import DummyResponse
+from astroquery.utils.tap.core import TapPlus
+from astroquery.utils.tap.xmlparser import utils
+from astroquery.vizier import Vizier
+
+from astroquery.esa.jwst import conf
+
+
+def data_path(filename):
+    data_dir = os.path.join(os.path.dirname(__file__), 'data')
+    return os.path.join(data_dir, filename)
+
+
+def get_plane_id_mock(url, *args, **kwargs):
+    return ['00000000-0000-0000-879d-ae91fa2f43e2'], 3
+
+
+@pytest.fixture(autouse=True)
+def plane_id_request(request):
+    try:
+        mp = request.getfixturevalue("monkeypatch")
+    except AttributeError:  # pytest < 3
```

Comment:
we are not supporting this old pytests, so you can drop the try/except, here and below, too

Reply:
Ok, done